README.md

```diff
@@ -1 +1,29 @@
-# To-do-testing
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+## To-do-testing
+
+In this code we use jest to make an automatic testing the code and show if the code work.
+the target of this project is to implement an automatic testing to know if your code is work with jest.
+## Description
+
+This is the first testing project with jest and simulated the html with jest-environment-jsdom to testing some many instance of the code.
+
+## Built with :
+
+- Html
+- Css
+- javaScript
+- jest (test tool)
+- webpack
+
```

Comment:
- [ ] As suggested by the previous reviewer, I kindly suggest that you add steps in your README file on how to run your scripts locally. This will help users know how to run your project.

## EXAMPLE
```
### Setup
If you installed git you can clone the code to your machine, or download a ZIP of all the files directly.

[Download the ZIP from this location](https://github.com/jdbs9514/To-do-testing/archive/refs/heads/main.zip), or run the following [git](https://git-scm.com/downloads) command to clone the files to your machine:

```bash
git clone https://github.com/jdbs9514/To-do-testing.git
```
```
- Once the files are on your machine, open the _todo-list_ folder in [Visual Studio Code](https://code.visualstudio.com/), and follow the steps in Installation & Set Up

## 🛠 Installation & Set Up

1. Install Node Modules

   ```sh
   npm install
   ```
```
2. Start the development server

   ```sh
   npm start
   ```
```
3. Open [http://localhost:8080](http://localhost:8080) to view it in the browser.

## 🚀 Building and Running for Production

1. Generate a full static production build

   ```sh
   npm run build
   ```
```
4. ## 🚀 Testing Application

1. Test the items Counter and Comments Counter function

   ```sh
   npm test
````